extensions/EXT/SPV_EXT_shader_atomic_float16_add.asciidoc

```diff
@@ -0,0 +1,138 @@
+SPV_EXT_shader_atomic_float16_add
+=================================
+
+Name Strings
+------------
+
+SPV_EXT_shader_atomic_float16_add
+
+Contact
+-------
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
+
+Contributors
+------------
+
+- Jason Ekstrand, Intel
+- Ben Ashbaugh, Intel
+
+Status
+------
+
+- Draft
```

Comment:
We should fix this to have the approval dates also, since this is no longer a `Draft`.

Reply:
Yup.  Fixed.  While I was fixing it, I looked at SPV_EXT_shader_atomic_float_add to see what to do and it was still marked draft too. :joy: